src/pages/api/sphere/webhook/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
maybe just do the api check here since the only place the webhook is going to send to is this endpoint and nowehre else

Reply:
Removed the middleware and added the HMAC body validation in API